docs/framework/backbone.md

```diff
@@ -0,0 +1,306 @@
+# Backbone Reference
+
+!!! failure "Deprecated"
+    CiviCRM no longer recommends using Backbone. This page is here primarily for archival purposes.
+
+## Background
+
+Backbone is a model-view (MV) framework for Javascript which follows a
+minimalist, plugin-oriented architecture: at its core, Backbone defines
+a minimal set of classes and utilities for building user-interfaces with
+object-oriented Javascript. Like the Drupal and jQuery communities, the
+Backbone community has a wide range of plugins to address missing
+functionality.
+
+## Examples
+
+Backbone is currently used in the following parts of CiviCRM:
+
+-   Profile Designer (v4.3+): Drag/drop interface for creating
+    profile forms. (At time of writing, usable with CiviSurvey)
+-   CiviVolunteer Extension (v4.4+): Drag/drop interface for managing
```

Comment:
CiviVolunteer still uses it too https://github.com/civicrm/org.civicrm.volunteer/blob/e97eeb88a6a10e8614dfa95ad530d6b6d8b93947/CRM/Volunteer/Form/Manage.php#L44 so satisfied this section is accurate